src/GraphQL.Tests/Bugs/Bug1781.cs

```diff
@@ -0,0 +1,95 @@
+using System.Numerics;
+using System.Threading.Tasks;
+using GraphQL.Language;
+using GraphQL.Types;
+using GraphQLParser.AST;
+using Shouldly;
+using Xunit;
+
+namespace GraphQL.Tests.Bugs
+{
+    // https://github.com/graphql-dotnet/graphql-dotnet/pulls/1781
+    public class Bug1781 : QueryTestBase<Bug1781Schema>
+    {
+        [Fact]
+        public async Task DocumentExecuter_really_big_double_valid()
+        {
+            var de = new DocumentExecuter();
+            var valid = await de.ExecuteAsync(new ExecutionOptions
+            {
+                Query = $"{{ test(arg:{double.MaxValue.ToString("0")}0.0) }}",
```

Comment:
What is the reason to use `0.0`?

Reply:
Also consider to use `:0` instead of `.ToString("0")`

Reply:
* `double.MaxValue.ToString("0")` would be intrepreted as an integer; this test is attempting to test floating point parsing
* This test is attempting to test floating point parsing where the result would be larger than `double.MaxValue`
Essentially adding `0.0` multiplies the value by 10 and then forces the parser to interpret it as a floating point value

Reply:
> Also consider to use `:0` instead of `.ToString("0")`

👍

Reply:
> Essentially adding 0.0 multiplies the value by 10 and then forces the parser to interpret it as a floating point value

It is worth to add comments about this.

Reply:
Done